docs/modules/reference/pages/service-assurance/monitors/CiscoPingMibMonitor.adoc

```diff
@@ -214,18 +217,23 @@ Example `poller-configuration.xml` additions:
     <rra>RRA:MIN:0.5:288:366</rra>
   </rrd>
   <service name="BarCentral" interval="300000" user-defined="false" status="on">
-    <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response" />
-    <parameter key="rrd-base-name" value="ciscoping" />
-    <parameter key="ds-name" value="ciscoping" />
-    <parameter key="proxy-ip-addr" value="20.11.5.11" />
-    <parameter key="target-ip-addr" value="$\{ipaddr}" />
+    <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response" /> <1>
+    <parameter key="rrd-base-name" value="ciscoping" /> <2>
+    <parameter key="ds-name" value="ciscoping" /> <3>
+    <parameter key="proxy-ip-addr" value="20.11.5.11" /> <4>
+    <parameter key="target-ip-addr" value="$\{ipaddr}" /> <5>
   </service>
   <downtime interval="30000" begin="0" end="300000" /><!-- 30s, 0, 5m -->
   <downtime interval="300000" begin="300000" end="43200000" /><!-- 5m, 5m, 12h -->
   <downtime interval="600000" begin="43200000" end="432000000" /><!-- 10m, 12h, 5d -->
   <downtime begin="432000000" delete="true" /><!-- anything after 5 days delete -->
 </package>
 
-<monitor service="BarCentral" class-name="org.opennms.netmgt.poller.monitors.CiscoPingMibMonitor" /> <1>
+<monitor service="BarCentral" class-name="org.opennms.netmgt.poller.monitors.CiscoPingMibMonitor" /> <6>
 ----
-<1> Required *monitor* section.
\ No newline at end of file
+<1> Base directory of an RRD repository in which to store this service monitor’s response-time samples.
+<2> The file name of the RRD file (minus the .rrd or .jrb file extension)
```

Comment:
Markdown isn't happy with the adoc format, the markdown gets semi lost when rendered [no gray background]. This will need to be resolved likely a different way.